src/dask_histogram/boost.py

```diff
@@ -112,11 +110,9 @@ def _histref(self):
         )
 
     def __iadd__(self, other):
-        if self.staged_fills() and other.staged_fills():
-            self._staged += other._staged
-        elif not self.staged_fills() and other.staged_fills():
-            self._staged = other._staged
-        return self
+        raise NotImplementedError(
```

Comment:
Could just remove add/iadd

Reply:
They are defined in the boost histogram base class, which would lead to some confusing behavior.

Reply:
ok